dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -400,19 +387,30 @@ internal class DokkaSymbolVisitor(
     )
 
     /**
-     * @param includeStaticScope flag to add static members, e.g. `valueOf`, `values` and `entries` members for Enum
+     * @return a scope [DokkaScope] consisting of:
+     * - primary and secondary constructors
+     * - member functions, including inherited ones
+     * - member properties, including inherited ones and synthetic java properties
+     * - classlikes (classes and objects **except a companion**) that are explicitly declared in [namedClassOrObjectSymbol]
+     *   only if [includeStaticScope] is enabled
+     *
+     * @param includeStaticScope a flag to add static members, e.g. `valueOf`, `values` and `entries` members for Enum.
+     * See [org.jetbrains.kotlin.analysis.api.components.KtScopeProvider.getStaticDeclaredMemberScope] for what a static scope is.
      */
     private fun KtAnalysisSession.getDokkaScopeFrom(
         namedClassOrObjectSymbol: KtNamedClassOrObjectSymbol,
         dri: DRI,
         includeStaticScope: Boolean = true
     ): DokkaScope {
         // e.g. getStaticMemberScope contains `valueOf`, `values` and `entries` members for Enum
```

Comment:
looks like this comment is outdated now, as `getStaticMemberScope` is used only for classifiers

Reply:
The comment is still actual.
I have replaced  `getMemberScope + getStaticMemberScope` with `getCombinedMemberScope` that do the same.

https://github.com/JetBrains/kotlin/blob/9fa0dfe4bc33b82b3d7d5bb775fae929eb151829/analysis/analysis-api/src/org/jetbrains/kotlin/analysis/api/components/KtScopeProvider.kt#L156
```kt
    public open fun getCombinedMemberScope(symbol: KtSymbolWithMembers): KtScope = 
    getCompositeScope(
        listOf(
            getMemberScope(symbol),
            getStaticMemberScope(symbol),
        )
    )
```
I can revert it if it is unobvious